lib/ansible/modules/system/cron.py

```diff
@@ -725,7 +725,7 @@ def main():
                 changed = True
 
     # no changes to env/job, but existing crontab needs a terminating newline
-    if not changed:
+    if not changed and not crontab.existing == '':
```

Comment:
I'm not sure of the difference between `not a == b` vs `a != b` for a negative equality check?